cardano-api/src/Cardano/Api/Query.hs

```diff
@@ -95,6 +96,8 @@ import           Data.SOP.Strict (SListI)
 import           Data.Text (Text)
 import qualified Data.Text as Text
 import           Data.Typeable
+import           Data.Word (Word64)
```

Comment:
Are you using linting with `stylish-haskell`?

Reply:
Yes, but I'm running it manually. I guess I have to add a git hook for that or something. Thanks for pointing out. I've reformatted.